<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Messenger</class>
 <widget class="QDialog" name="Messenger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Messenger</string>
  </property>
  <widget class="QListWidget" name="contactView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>181</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>490</y>
     <width>671</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>490</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QListWidget" name="chatView">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>761</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="contactButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add contact</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="serverEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="portEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Server</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="accountEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Account</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>50</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Guest</string>
   </property>
  </widget>
  <widget class="QListWidget" name="stateView">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>40</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>serverEdit</tabstop>
  <tabstop>portEdit</tabstop>
  <tabstop>accountEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>connectButton</tabstop>
  <tabstop>contactView</tabstop>
  <tabstop>contactButton</tabstop>
  <tabstop>inputEdit</tabstop>
  <tabstop>sendButton</tabstop>
  <tabstop>chatView</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>stateView</sender>
   <signal>indexesMoved(QModelIndexList)</signal>
   <receiver>stateView</receiver>
   <slot>scrollToBottom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>815</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>815</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
